### PR TITLE
Support for jmeter 2.11, 2.12, 2.13

### LINKED ISSUE #36: jmeter 2.11 ++ support
Do you plan to support jmeter 2.11++ versions?

### LINKED ISSUE #40: Jmeter stderr is not captured anywhere
Jmeter is launched as a separate process 

    ProcessBuilder processBuilder = new ProcessBuilder(createArgumentList(specs, workingDirectory))

The stdout (and stderr) of this process is not captured anywhere. If an error occurs before jmeter is launched completely, then jmeter.log is not created and the launch error is lost, making debugging challenging.

**workaround**: redirect stdout to parent process (gradle's) stdout.

    ProcessBuilder processBuilder = new ProcessBuilder(createArgumentList(specs, workingDirectory)).inheritIO()

### PR DESCRIPTION
**Pull request by foragerr**
Closes kulya/jmeter-gradle-plugin#36
Closes kulya/jmeter-gradle-plugin#40

### CODE REVIEW

**Comment by kulya** [comment]
Versions 1.3.3-2.xx released to maven central repositories. It will take few hours to propagate them to maven central. If you need it sooner, please specify additional maven repo in your build.gradle "https://oss.sonatype.org/content/groups/public"

**Comment by foragerr** [comment]
Thanks @kulya That was quick!

File: build.gradle [hunk 1]
```diff
@@ -25,31 +23,66 @@ repositories {
...
 
 task wrapper(type: Wrapper) {
-    gradleVersion = '1.10'
+    gradleVersion = '2.3'
+}
+
+configurations {
+	//workaround for jmeter defect 57555 (https://bz.apache.org/bugzilla/show_bug.cgi?id=57555)
+	if (jmeterVersion == '2.12'){
+		compile.exclude group: 'excalibur-fortress'
```
Comment (foragerr) [inline build.gradle]: These excludes don't make it to pom files generated with the maven plugin. Need to add 'module:' information as well.

**Comment by foragerr** [comment]
@kulya I just checked the binaries at oss.sonatype, looks the transitive dependency excludes didn't go into the pom files. Created issue #41

**Comment by djangofan** [comment]
Yeah, I have trouble starting it as version 2.13.  Cant find commons-math3 and commons-pool2  .   When it try to start as 2.12, the error is different: Could not find d-haven-managed-pool:d-haven-managed-pool:1.0

**Comment by foragerr** [comment]
fixed in #42

**Comment by djangofan** [comment]
After great difficulty, I finally got Gradle to load Jmeter 2.13  (with help from your changes)  and I was also able to get all the jmeter-plugins  to load ,  as well as   groovy-all, opencsv, guava, commons-lang3   .   Here is my build script:  https://github.com/djangofan/launch-jmeter/blob/master/build.gradle

**Comment by foragerr** [comment]
@djangofan let's move this conversation here: https://github.com/djangofan/launch-jmeter/issues/1

**Pull request merged**